code/ARAX/ARAXQuery/operation_to_ARAXi.py

```diff
@@ -10,12 +10,20 @@ def _make_expand_query(KP_name, qedge_keys):
     araxi_string = ""
     if not KP_name:
         araxi_string = "expand("
-    else:
-        araxi_string = f"expand(kp={KP_name}"
+
+    elif isinstance(KP_name, list):
+        if len(KP_name) > 0:
+            araxi_string = re.sub("['\"]", "",f"expand(kp={KP_name}")
```

Comment:
I'm trying to parse the "['\"]", and see the lone single quote: does this require the KP_name to use specific quote types and/or omit spaces?